ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -25,14 +34,28 @@ type OwnProps = {
   navigation: NavigationScreenProp<NavigationState>;
 };
 
-type Props = ReturnType<typeof mapStateToProps> & ReduxProps & OwnProps;
+type Props = ReturnType<typeof mapStateToProps> &
+  OwnProps &
+  ReturnType<typeof mapDispatchToProps>;
+type RequestState = pot.Pot<"LOADING" | "LOADING_COMPLETE", string>;
```

Comment:
mmh this doesn't make much sense as `Pot` has already a `loading` state?

what's the meaning of `pot.someLoading<LOADING>("LOADING")` ?

Reply:
@cloudify 

interface SomeLoading<T> {
    readonly kind: "PotSomeLoading";
    readonly value: T;
}

I had to provide a value to someLoading function. I think pot is thought to carries data when loading. In this case we have to take track mainly of the state that has not value associated.

How can I improve this?

Reply:
use something simple like `true` then, to avoid confusion:

`pot.Pot<true, string>`

```ts
pot.none; // initial state
pot.noneLoading; // loading state
pot.some<true>(true); // loading completed state
```